In [ ]:
def binference(prior, givenpdisease, lpositive):
  posterior = (givenpdisease * prior) / lpositive
  return posterior

prior = float(input("Enter the prior probability of the disease: "))
givenpdisease = float(input("Enter the likelihood of a positive test given the disease: "))
lpositive = float(input("Enter the overall probability of a positive test: "))

posterior = binference(prior, givenpdisease, lpositive)
print(f"Posterior probability of disease positive test: {posterior}")

Enter the prior probability of the disease: 0.01
Enter the likelihood of a positive test given the disease: 0.9
Enter the overall probability of a positive test: 0.05
Posterior probability of disease positive test: 0.18000000000000002


3. Feedback

This means that after a positive test, the probability of actually having the disease is 18%, which is much higher than the prior probability but still indicates a significant chance of not having the disease.